### PR TITLE
Add a podspec

### LINKED ISSUE #10: Add CocoaPods support

### PR DESCRIPTION
**Pull request by sharplet**
Currently `pod lib lint` passes but `pod spec lint` would fail because the tag doesn't exist yet.
- [x] Lint podspec on CI

Fixes #10.

### CODE REVIEW

**Comment by jakecraige** [comment]
👍

**Comment by sharplet** [comment]
@jakecraige I could use a re-review now that I've added a bunch of changes for testing CocoaPods on CI.

File: circle.yml [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
   xcode:
-    version: 8.0
+    version: 7.3
   environment:
     NSUnbufferedIO: YES
 
 dependencies:
   override:
+    - gem install cocoapods
+    - pod install
```
Comment (jakecraige) [inline circle.yml:10]: Might be nice to have these in a `bin/setup` so contributors can run the same setup locally, thoughts?

Comment (jakecraige) [inline circle.yml:10]: Oh I just saw your comment, nvm. Since this is CI only I'm fine with it here

**Comment by jakecraige** [comment]
Cool cool 👍

**Pull request merged**